src/EA.Weee.Core/AatfReturn/AatfAddressData.cs

```diff
@@ -6,7 +6,7 @@
 
     public class AatfAddressData : AddressData
     {
-        [Required]
+        [Required(ErrorMessage = "Enter AATF/ATF site name")]
```

Comment:
Can we create unit tests for these attributes on this model as its an annoying one, and if we change them and test fails at least we should ask ourselves should we bit changing

Reply:
OK done